### PR TITLE
Repositions and styles 'new posts' button

### PR DESCRIPTION
**Pull request by itsalwaysbenny**
'New posts' link now appears in a fixed position beneath the nav bar at the top of the page. Changes colour upon mouse hover.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (BertZZ) [inline app/views/posts/index.html.erb]: Why are these 2 lines here?

**Review by BertZZ — CHANGES_REQUESTED** [review]
Looks fine to me however you seem to have some failing Rspec tests. It seems like you have broken the Routing so maybe @TimRobinson1 is the perfect person to check this out

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (will-gant) [reply app/views/posts/index.html.erb]: Deleted

**Comment by will-gant** [comment]
@BertZZ I've fixed the failing tests, which were checking that the button *doesn't* appear when you're not logged in.

**Comment by BertZZ** [comment]
Cool beans. Send it to @neoeno for a look

**Review by BertZZ — APPROVED** [review]

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -168,3 +168,21 @@ nav {
...
+  letter-spacing: 1px;
+  font-size: 14px;
+  position: fixed;
+  min-width:100%;
+  margin-top: 80px;
+}
+
+.new-posts:hover {
+    background: grey;
+}
```
Comment (neoeno) [inline app/assets/stylesheets/application.css.scss]: I'd probably move these classes to their own file — `application.css` can often grow out of control if it is added to often! Maybe `navigation.css`, or something similar.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -27,11 +27,18 @@
...
     <% else %>
     <%= link_to 'Sign in', sign_in_path %>
     <% end %>
+
   </nav>
-  <div class="header-buffer"></div>
+
+
+  <div class="header-buffer">
+  </div>
```
Comment (neoeno) [inline app/views/layouts/application.html.erb]: I'd like to see this diff be just lines 30-32 — the whitespace and formatting changes seem to add churn without benefit.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -27,11 +27,18 @@
     <a href="/" class="right-link" >Home</a>
     <%= image_tag current_user.avatar_url, size: User::AVATAR_SIZES[:small], class: "right-link" if current_user.avatar? %>
     <span class="right-link username"><%= current_user.name %></span><span class="right-link">Signed in as: </span>
+    <%= link_to new_post_path, class: "new-posts" do %>
+      New post
+    <% end %>
```
Comment (neoeno) [inline app/views/layouts/application.html.erb:32]: Nice, I like this `link_to do` syntax!

**Review by neoeno — CHANGES_REQUESTED** [review]
Nice work! Just a few small comments

**Comment by will-gant** [comment]
@neoeno - thanks for the review! Have extracted the CSS as you suggested, and also *tried* to reformat the changes to layout so the change is limited to lines 30-32. Hope it's OK.

**Review by neoeno — APPROVED** [review]

**Comment by neoeno** [comment]
Looks good! Great work here!

**Comment by BertZZ** [comment]
I think I might have accidently broke this, Looks like Ill have to try fix it

**Pull request merged**